install.sh

```diff
@@ -338,19 +338,28 @@ do_download () {
 
     if [[ -z "${wget_rc}" ]]; then
         echo "Downloading ${url}..."
+        local header
+        if [[ -n "${RS_USER_AGENT}" ]]; then
+          header="-d --header 'User-Agent: ${RS_USER_AGENT}'"
```

Comment:
Just double checking, did you want the `-d` debug flag in here? I noticed a similar flag wasn't added for curl.

Reply:
Ah, thanks for that catch!  I've removed it.